### ISSUE TITLE
[Feature Request]: Firendly fire option 

### ISSUE THREAD

**Issue by BaptisteII** [FTBTeam/FTB-Mods-Issues#984]
### Mod

FTB Teams

### Description

I noticed that the friendly fire option was not present in the mod I think it would be wise to put it.

### Describe the solution you'd like

In the administrator options (attached screen) there would be a "friendly fire" section that could be set to true or false for all teams.

![image](https://github.com/FTBTeam/FTB-Mods-Issues/assets/147993029/1b8bc6a0-c1e3-4c9d-95d1-da04bd3896ed)

**Comment by desht** [comment]
This might be better done by some integration with vanilla scoreboard teams (see https://github.com/FTBTeam/FTB-Mods-Issues/issues/631 for more discussion). We can take advantage of existing vanilla PvP protection that way.

**Comment by BaptisteII** [comment]
Yeah it's a good Idea but it need an op too create the team and i think it's more simply if it's integrated in ftb

**Comment by desht** [comment]
Not necessarily, if FTB Teams creates the scoreboard team on behalf of the FTB team. So it would be integrated, in that respect.

**Comment by BaptisteII** [comment]
Oh well i understand i think. You want to say this is ftb who create the scoreboard ?

**Comment by HeavyDeception** [comment]
Hi wanted to ask if this was a feature that was going to be added but put on the back burner? I know that Minecraft has the Teams command but normal players can't use it so I was just wanting to see what was happening.

**Comment by manmaed** [comment]
Please disregard the previous reply from the stale bot, it's been a bad bot and sent for recycling.
!skip

**Comment by Midlex** [comment]
bump, unless there's an alternative im unaware of this would be the perfect option for most smaller servers

**Comment by bartico6** [comment]
Another bump, mainly because it's surfaced again (we're oneshotting each other with iron's spells by accident 😭)
